changes/796.feature.md

```diff
@@ -0,0 +1,3 @@
+Added a total_length() function to hikari.Embed
+- Takes into account the character length of the embed's title, description, fields (all field names and values), footer, and author combined.
+- Useful for determining if the embed exceeds Discord's 6000 character limit.
```

Comment:
Again, not worth documenting the character limit

Reply:
Sane here, changelogs are essentially frame so any data referenced in it (or any changes for that matter) shouldn't have to stay up to date anyways

Reply:
Will leave it in that case